example/routing.ts

```diff
@@ -23,4 +24,10 @@ export const routing: Routing = {
       upload: uploadAvatarEndpoint,
     },
   },
+  // path /public serves static files from /example/assets
+  public: serveStatic(path.join(__dirname, "assets"), {
```

Comment:
perhaps it would be better to make it a class in order to preserve the consistency with `new DependsOnMethod`